### ISSUE TITLE
Viewing all teams on gettogether.community results in an error.

### ISSUE THREAD

**Issue by ghost** [GetTogetherComm/GetTogether#327]
https://gettogether.community/teams/all/

"
Internal Server Error

The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator at [no address given] to inform them of the time this error occurred, and the actions you performed just before this error.

More information about this error may be available in the server error log.
Apache/2.4.18 (Ubuntu) Server at gettogether.community Port 443
"

Basically just click "Teams" near the upper-left, and then click "All Teams" near the center-top.

My account is pretty new (since I'm locked out of my old one (forgot password, dead e-mail)) so it may be related to that.
